services/frontend/pkg/revaconfig/config.go

```diff
@@ -9,7 +10,15 @@ import (
 )
 
 // FrontendConfigFromStruct will adapt an oCIS config struct into a reva mapstructure to start a reva service.
-func FrontendConfigFromStruct(cfg *config.Config) map[string]interface{} {
+func FrontendConfigFromStruct(cfg *config.Config) (map[string]interface{}, error) {
+
+	webURL, err := url.Parse(cfg.PublicURL)
+	if err != nil {
+		return map[string]interface{}{}, err
+	}
+	webURL.Path = "/external"
```

Comment:
Wouldn't `webURL.Path = path.Join(webURL.Path, "/external")` be better since the PublicURL could theoretically already contain a path like `https://example.com/myocisinstance`?